impl/src/bitfield.rs

```diff
@@ -271,22 +325,96 @@ impl BitfieldStruct {
     /// amount of bytes to compactly store the information of all its bit fields.
     fn generate_struct(&self) -> TokenStream2 {
         let span = self.item_struct.span();
-        let attrs = &self.item_struct.attrs;
+        let attrs = self.get_attrs_no_debug();
         let vis = &self.item_struct.vis;
         let ident = &self.item_struct.ident;
         let size = self.generate_bitfield_size();
         let next_divisible_by_8 = Self::next_divisible_by_8(&size);
         quote_spanned!(span=>
             #( #attrs )*
             #[repr(transparent)]
-            #[allow(clippy::identity_op)]
+            #[allow(clippy::identity_op, unused_attributes)]
```

Comment:
why is `unused_attributes` needed here? what code spawns which warnings?

Reply:
It's needed for empty derives. So `#[derive(Debug)]` becomes `#[derive()]` (which raises the warning). If you have strong feelings about going out of my way to detect and remove this though, I can handle it that way.

Reply:
Would it be possible to simply check if no derives are left and therefore skip re-expanding the whole derive attribute again?